pilot/pkg/security/authn/v1beta1/policy_applier.go

```diff
@@ -188,6 +188,9 @@ func (a *v1beta1PolicyApplier) InboundMTLSSettings(
 		mc = a.push.Mesh
 	}
 	// Configure TLS version based on meshconfig TLS API.
+	// This is used to configure TLS version for inbound filter chain of ISTIO MUTUAL cases.
+	// For MUTUAL and SIMPLE TLS modes specified via ServerTLSSettings in Sidecar or Gateway,
```

Comment:
The above code is picking up from tlsDefaults as well, if not specified in ServerTLSSettings?

Reply:
The new code I have added in BuildListenerContext does it

Reply:
yeah, was just asking if that has to be specified in the comment. But anyways you have mentioned the func name. Just a minor comment, in the API docs, I had added that only ecdhCurves is currently supported via tlsDefaults :D  https://preliminary.istio.io/latest/docs/reference/config/istio.mesh.v1alpha1/ may be we can change it once PR is merged